tools/run_tests/run_interop_tests.py

```diff
@@ -226,11 +229,15 @@ class Http2Client:
   """
   def __init__(self):
     self.client_cwd = None
+    self.server_cwd = None
     self.safename = str(self)
 
   def client_cmd(self, args):
     return ['tools/http2_interop/http2_interop.test', '-test.v'] + args
 
+  def server_cmd(self, args):
```

Comment:
I think that the http2 server should have its own class. It doesn't really make sense to push it into the Http2Client class.